test_conformance/computeinfo/main.cpp

```diff
@@ -736,7 +736,7 @@ void dumpConfigInfo(config_info* info)
             }
             break;
         case type_cl_device_id:
-            log_info("\t%s == %ld\n", info->opcode_name,
+            log_info("\t%s == %" PRIdPTR "\n", info->opcode_name,
                      (intptr_t)info->config.device_id);
```

Comment:
For my own education, is there any reason we don't use `%p` here?

Reply:
I was wondering the same, but decided not to change the output format as part of this PR (`%p` will print in hexadecimal, while the test currently outputs decimal).